components/collapsible-view/views/collapsible-view.js

```diff
@@ -0,0 +1,58 @@
+define(['backbone', 'underscore'],
```

Comment:
Nit: I was thinking of just ``components/views/collapsible-view.js``. I don't like lots of single file directories.